### ISSUE TITLE
TicTacToe game cannot reach stalemate end.

### ISSUE THREAD

**Issue by RileyFitz** [RileyFitz/terminal-games#1]
With the TicTacToe game, it is impossible to reach an end state for a singular game. This does not hurt the requirements of embeded TTT, so I am moving past it.

I suspect this is a similar issue to the 'xTurn' change issue. The change to 'xTurn' would not apply when executed inside the function. 
This was mitigated by being performed inside the initiate_game function. 

It cannot termiante bc there are no valid moves after all positions are taken, and the loop continues without quiting for some reason.

tldr;
Games cannot end in a stalemate, unsure why self value changes and While loop is not broken.

**Comment by RileyFitz** [comment]
Changed both ttt and embedded_ttt board states to be a singular array instead of a nested array. The ensuing changes corrected this bug.
